erpnext_documentation/www/docs/user/manual/en/support/issue.md

```diff
@@ -32,10 +32,13 @@ replied, its status becomes "Replied".
     * Open: The Issue has been created and yet to be replied to.
     * Replied: A reply has been sent to the Issue.
     * Hold: The Issue is on Hold due to some reason.
+    * Resolved: When the users are reasonably sure that they have provided the customer with a solution to their problem but have not acknowledgement about the resolution from the customer.
     * Closed: The Customer got a resolution and the Issue was closed.
```

Comment:
The Customer got a satisfactory resolution that he has indicated with an acknowledgment and the Issue was closed.